services/api/src/typeDefs.js

```diff
@@ -1415,9 +1415,9 @@ const typeDefs = gql`
     """
     Get an organization by its ID
     """
-    organizationById(id: Int!): Organization
+    organizationById(organization: Int!): Organization
```

Comment:
This must have crept in through the merge conflict

Reply:
Need to go back to `id`

Reply:
Oops, good catch! Fixed now